src/components/nav-menu-item/nav-menu-item.tsx

```diff
@@ -281,6 +330,37 @@ export class CalciteNavMenuItem {
     this.editingActive = !this.editingActive;
   };
 
+  private handleMenuItemSlotChange = (event: Event): void => {
+    if (this.hasSubMenu) {
+      this.subMenuItems = slotChangeGetAssignedElements(event) as HTMLCalciteNavMenuItemElement[];
+    }
+  };
+
+  private hasSlottedItems(): boolean {
+    return this.el.querySelectorAll("[slot=menu-item-dropdown]").length > 0;
+  }
+
+  private focusFirst(): void {
+    setTimeout(() => this.subMenuItems[0].setFocus(), 1000);
+  }
```

Comment:
Think I recall additional R&D is needed on the focus - wanted to point it out again, just to make sure. The delay is a bit disruptive to AT, where if the user tries to press any key, focus is completely lost.